docs/installation/configuration/opennotifs_config.rst

```diff
@@ -55,21 +53,72 @@ using Open Zaak as the example.
-      - **Secret**: *The same Secret as given in Open Zaak step 2c*
+      - **Client ID**: *For example:* ``open-zaak``
+      - **Secret**: *Some random string*
 
       Make sure Open Zaak is configured to use this Client ID and secret to
       access Open Notificaties.
 
    d. Click **Opslaan**.
 
+3. After that, we need to configure the **Notificatiescomponentconfiguratie**.
+
+   a. Navigate to **Configuratie > Notificatiescomponentconfiguratie** and add a new **Notifications api service**
+   b. Fill out the form and use the client ID and secret from step 1c
+   c. Click **Opslaan** on the **Service** form
+   d. Click **Opslaan** on the **Notificatiescomponentconfiguratie** form
+
+4. Finally, we need to add the client ID and secret from step 1c under **API Autorisaties > Autorisatiegegevens > Autorisatiegegeven toevoegen**
+
+Configure Open Zaak
+===================
+
+In case Open Zaak is not configured yet, refer to `configuration of Open Zaak`_. Use the following as input for the management command:
+
+   - **notifications-api-app-client-id**: client ID from step 1c above
+   - **notifications-api-app-secret**: secret from step 1c above
+   - **notifications-api-client-id**: client ID from step 2c above
+   - **notifications-api-secret**: secret from from step 2c above
+
+Add notification consumers
+==========================
+
+In order to add new consumer webhooks that are subscribed to specific channels, follow these steps:
+
+1. Ensure your consumer webhook can handle the notification body (see `API specification`_)
+2. Ensure your consumer webhook responds to notifications with a status code in the range 200-209
+3. (Optional) If no **Kanaal** exists for the resource, create it via the admin interface
+
+   a. Navigate to **Notificaties** > **Kanalen** > **Kanaal toevoegen**
+   b. Fill out the form:
+
+      - **Naam**: the name of the channel
+      - **Documentatie link**: optional, URL that points towards documentation about the channel
+      - **Filters**: optional, comma separated list of attributes on which subscriptions can filter
+
+4. Configure an **Abonnement** for the consumer webhook
+
+   a. Navigate to **Notificaties > Abonnementen > Abonnement toevoegen**
+   b. Fill out the form:
+
+     - **Callback URL**: the URL of the consumer webhook to which Open Notificaties will send notifications
+     - **Autorisatie header**: the authorization header that Open Notificaties will use for its requests to the callback URL
+     - **Client ID**: optional, the client identifier used in the authorization header (in case of JWT)
+     - **Filters > Kanaal**: the channels to which the consumer webhook should be subscribed
+
+   .. warning:: The `manual`_ describes how to add **Abonnementen**, but the mechanism to support authentication (JWT or otherwise) does not function properly at the moment. The method to add **Abonnementen** as described above is a workaround for this issue
```

Comment:
Created an issue for this https://github.com/open-zaak/open-zaak/issues/1694

Reply:
Commented there :)